integrations/wp-importer/wp-import.php

```diff
@@ -128,35 +128,40 @@ protected function remap_terms_relations( &$terms ) {
 			$translations = maybe_unserialize( $term['term_description'] );
 			foreach ( $translations as $slug => $old_id ) {
 				if ( $old_id && ! empty( $this->processed_terms[ $old_id ] ) && $lang = PLL()->model->get_language( $slug ) ) {
-					// Language relationship
-					$trs[] = $wpdb->prepare( '( %d, %d )', $this->processed_terms[ $old_id ], $lang->get_tax_prop( 'term_language', 'term_taxonomy_id' ) );
+					// Language relationship.
+					$trs[] = array( $this->processed_terms[ $old_id ], $lang->get_tax_prop( 'term_language', 'term_taxonomy_id' ) );
 
-					// Translation relationship
-					$trs[] = $wpdb->prepare( '( %d, %d )', $this->processed_terms[ $old_id ], get_term( $this->processed_terms[ $term['term_id'] ], 'term_translations' )->term_taxonomy_id );
+					// Translation relationship.
+					$trs[] = array( $this->processed_terms[ $old_id ], get_term( $this->processed_terms[ $term['term_id'] ], 'term_translations' )->term_taxonomy_id );
 				}
 			}
 		}
 
-		// Insert term_relationships
+		// Insert term_relationships.
 		if ( ! empty( $trs ) ) {
-			$trs = array_unique( $trs );
-
-			// Make sure we don't attempt to insert already existing term relationships
+			// Make sure we don't attempt to insert already existing term relationships.
```

Comment:
`$trs = array_unique( $trs );` has been lost. Is it on purpose?

Reply:
I don't remember. I guess I found  it's useless and thus removed it.